src/ucp/api/ucp.h

```diff
@@ -1677,6 +1678,11 @@ typedef struct {
      */
     ucs_memory_type_t memory_type;
 
+    /**
```

Comment:
@yosefe for ABI compatibility, this field has to go to the end of structure, right ?

Reply:
yes

Reply:
done